Makefile

```diff
@@ -834,6 +834,9 @@ cache-export-docker-load-all: $(LINUXKIT) $(addsuffix -cache-export-docker-load,
 proto-vendor:
 	@$(DOCKER_GO) "cd pkg/pillar ; go mod vendor" $(CURDIR) proto
 
+bump-eve-api:
```

Comment:
@christoph-zededa , I forgot to mention that would be good to have this target explained in the `make help` target....

Reply:
done; please have a look at the help text